docs-api/frontity-plugins/embedded-mode.md

```diff
@@ -0,0 +1,69 @@
+# Frontity Embedded Mode
+
+This plugin enables an **Embedded Mode** implementation of Frontity. It can be downloaded from [here](https://github.com/frontity/frontity-embedded).
+
+Embedded Mode can be contrasted with **Decoupled Mode**. In Decoupled Mode the primary domain points to the Frontity site, with the WordPress site being on a secondary domain or on a subdomain of the primary domain.
+
+In Embedded Mode the primary domain points to the WordPress site, and the Frontity site can be on another domain including the default URL provided by a free hosting service such as [Vercel](https://docs.frontity.org/deployment/deploy-using-vercel).
+
+This plugin replaces the active WordPress theme with the Frontity installation. Frontity therefore effectively becomes the WordPress sites' theme. It works by substituting it's own [`template.php`](https://github.com/frontity/frontity-embedded/blob/master/includes/template.php) in place of any call made to the [WordPress template hierarchy](https://developer.wordpress.org/themes/basics/template-hierarchy/).
+
+## Advantages of Embedded Mode
+
+Embedded Mode offers several advantages over Decoupled Mode.
+
+- The WordPress site exists under the primary domain. This ensures that:
+  - the SEO of the site is not compromised
+  - sitemaps generated by WordPress/Yoast/All-in-one-SEO work as normal
+  - caching plugins continue to work as normal
+- Content producers/editors continue to have the same experience
+  - post/page preview remains available
+  - the admin bar is active for logged in users
+
+## Disdvantages of Embedded Mode
+
+- Frontity still needs to be hosted on a separate node server/serverless function (albeit on any domain you like including default domains provided by the hosting provider)
+- WordPress still needs to go through it's [bootstrap process](https://wordpress.tv/2017/06/22/alain-schlesser-demystifying-the-wordpress-bootstrap-process/) on initial page load
+- more routing is involved (WordPress makes a call to Frontity which returns the HTML to WordPress) so a caching plugin is a necessity rather than simply a nice to have. (We recommend [Simple Cache](https://github.com/luisherranz/simple-cache))
```

Comment:
Simple Cache is broken when used along with the REST API, so we either have to explain that they need to exclude `/wp-json` or publish my version of Simple Cache, which does have support for the REST API.

@santosguillamot let's make a decision about that here: https://community.frontity.org/t/product-feature-freeze-the-framework/4347/31

Reply:
Okay, I've answered you in the thread, but I think it makes sense to publish the plugin in the repository :slightly_smiling_face:

Reply:
I'll update the link once it's been accepted and published in the repository.